obs-packaging/gen_versions_txt.sh

```diff
@@ -73,7 +73,7 @@ qemu_lite_version=${qemu_lite_version}
 qemu_lite_hash=${qemu_lite_hash}
```

Comment:
please fix commit message: The qemu vanilla sha *was was* incorrect.

Reply:
@devimc , thanks. Fixed now.